### ISSUE TITLE
The predicted edge line is too thick

### ISSUE THREAD

**Issue by wangxiang-wx** [MengyangPu/EDTER#83]
Thank you for sharing your great job !
When I used my own dataset for training, I found that the edge lines of the test were too thick. What is the reason for this problem? Thank you for your guidance and suggestions

**Comment by MengyangPu** [comment]
@wangxiang-wx 
Hi, could you share more details? e.g., the size of your image, the sample image and its visualization result?

**Comment by wangxiang-wx** [comment]
Ok，Thank you for your attention to my question
（1）Configuration
GPU=nvidia 3080Ti  memory=16GB
（2）Training settings
lr=1e-6  iteration=80k   The training hyper-parameters and the hyper-parameters in the code are exactly the same
（3）dataset
the size of train img=(1280，720)，the size of test img=(481，321)，Pixel width of the label=1
The purpose of the experiment is to be able to recognize edge lines in such earthwork scenes。
![1](https://github.com/MengyangPu/EDTER/assets/67371650/308f697b-a81c-49f5-83e6-d2c05f7d346e)
![2](https://github.com/MengyangPu/EDTER/assets/67371650/98a0f9a6-777f-4b1a-9808-8ca1393c6089)
![3](https://github.com/MengyangPu/EDTER/assets/67371650/e262ab05-0743-41cb-a0f2-4d86f3aebc37)

The following are training (testing) samples, ground truth, and visualization results attached
I'm not sure what the reason is. After testing, I found that all test results are visualized like this. Although the edge line detection position is correct, this line is too thick, so the accuracy AP is also very low！
Thank you for your guidance, thank you again!!!!

**Comment by MengyangPu** [comment]
@wangxiang-wx 
First, the edge output by the convolutional network is thicker. 
I have several suggestions for your question:
1. Change training parameters such as learning rate, batch size, and number of iterations. It is necessary to change the training parameters for different datasets.
2. Why do the input and output images have different sizes? Maybe this is one of the problems.
3. In our experiments, batch size=8. Please make sure your batch size is not less than 8.

**Comment by wangxiang-wx** [comment]
Thank you for your advice！Now I have solved this problem and have a high AP and other evaluation indicators. Thank you again for your work and suggestions!

**Comment by MengyangPu** [comment]
You're welcome! It's great to know that my advice was helpful.
